ci/install_houdini2.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
-# move hou tarball into top-level and untar
-cp hou/hou.tar.gz .
-tar -xzf hou.tar.gz
+if [ -d "hou" ]; then
+    # move hou tarball into top-level and untar
+    cp hou/hou.tar.gz .
+    tar -xzf hou.tar.gz
+fi
```

Comment:
This is also for fork repos, where the job will no longer fail if Houdini has not been downloaded, instead it'll silently succeed. It should be possible to create a fork of OpenVDB and make a change to the core library with successful GitHub Actions workflows running without needing to first setup Houdini installs.